### DISCUSSION TITLE
How to rename the EVO manager folder in EVO2?

### DISCUSSION THREAD

**Discussion by q2apro** [evolution-cms/evolution#1910] [Q&A]
After the [update from EVO1 to EVO2](https://github.com/evolution-cms/evolution/issues/1745#issuecomment-755885137) I wanted to rename the `manager` folder (like in EVO1) to have more security, so I:

- edited file `/assets/cache/siteManager.php` and added this line:

```
<?php
define('MGR_DIR', 'managerhidden');
```

- then I renamed the folder `manager` to `managerhidden`
- but now when I navigate to `/managerhidden` it throws an error that other files in `manager` cannot be found

> PHP Fatal error:  require_once(): Failed opening required '/manager/includes/protect.inc.php' (include_path='.:/opt/sp/php7.4/lib/php') in /index.php on line 69

So the question is: How to rename the EVO manager folder in EVO2?

**Comment by Dmi3yy** [comment]
file:  /core/custom/define.php

define('MGR_DIR', 'manager2’);


and 
 
file: config.php in root folder  

return [
    'manager' => __DIR__ . '/manager2',
];

**Comment by q2apro** [comment]
What path has the "root folder"?

Maybe you mean file `config.inc.php` in `\manager\includes\`?

**Comment by Dmi3yy** [comment]
Near with index.php

**Comment by q2apro** [comment]
I created a file `config.php` in the root (next to the main `index.php`). Content:

```
<?php

return [
	'manager' => DIR.'/manager2',
];
```

When opening the website now, it throws an error:

> Warning: Use of undefined constant DIR - assumed 'DIR' (this will throw an Error in a future version of PHP) in /config.php on line 4

And the website does not work anymore.

Can you please tell the right path of the config.php file? Thanks.

**Comment by Dmi3yy** [comment]
Bad copypast 

файле /core/custom/define.php

define('MGR_DIR', 'manager2’);
и 
config.php в root 
`
return [
    'manager' => __DIR__ . '/manager2',
];
`

**Comment by q2apro** [comment]
1. Created a file `/core/custom/define.php`
2. Created a file `/config.php`
3. Uploaded both files to the server
4. Renamed manager to manager2

When opening \manager2 I get the error message:

> Unable to load configuration settings
> Please run the Evolution CMS install utility

It is not working.

**Comment by Dmi3yy** [comment]
In core config need use 3 params: like in sample

https://github.com/evolution-cms/evolution/blob/3.x/config.php.example

and all will work 

tested
